src/GraphingInterfaces/IGraphRenderer.h

```diff
@@ -21,5 +26,7 @@ namespace Graphing::Renderer
         virtual HRESULT ChangeRange(ChangeRangeAction action) = 0;
         virtual HRESULT MoveRangeByRatio(double ratioX, double ratioY) = 0;
         virtual HRESULT ResetRange() = 0;
+
+		virtual HRESULT GetBitmap(std::shared_ptr<MathSolverEngine::Graph::Renderer::IBitmap>& bitmapOut, bool& hasSomeMissingDataOut) = 0;
```

Comment:
This should also be from the Graphing namespace

Reply:
Fixed